docs/en/setup/service-agent/java-agent/Supported-list.md

```diff
@@ -73,6 +73,9 @@
 * Vert.x Ecosystem
   * Vert.x Eventbus 3.2+
   * Vert.x Web 3.x
+* Multi-threaded Framework
```

Comment:
Thread Schedule Framework

I think we could add more threadpool plugin in this catalog.

Reply:
ok